test/template-service-test.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+// Disable self signed testing for the tests
+var service = new (require('../service/template-service'))();
+var fs = require('fs');
+
+var templateModels = {
+  validTemplate: './test/template/test-email.html',
+  invalidTemplate: './test/template/' + global.testUid() + '/test-email.html',
+  templatePath: './test/template',
+  loadedTemplates: ['test-email.html'],
+  createTemplate: './test/template/' + global.testUid() + '-email.html',
+  templateBody: '<h1>Hello {{name}}</h1>',
+  updateTemplateBody: '<h1>Hello {{name}}</h1>This is updated',
+};
+
+describe('The service should be initialised', function serviceInitialization() {
```

Comment:
Fair enough, but due to pedantry I am linting the tests as well to remind myself the `correct` way to do this